iothub-manager/app/com/microsoft/azure/iotsolutions/iothubmanager/services/external/ConfigurationsHelper.java

```diff
@@ -63,10 +63,12 @@ public static Configuration toHubConfiguration(final DeploymentServiceModel mode
 
         // Required labels
         labels.put(PACKAGE_TYPE_LABEL, model.getPackageType().toString());
```

Comment:
Should we check for empty and null values in other labels?

Reply:
As per our discussion, we are checking in the service/controller layer, this is a helper function, so I don't think we need this.